src/components/victory-line.jsx

```diff
@@ -89,6 +90,11 @@ export default class VictoryLine extends React.Component {
      */
     label: React.PropTypes.string,
     /**
+     * The labelComponent prop takes in an entire, HTML-complete label component
+     * which will be used to create labels for line to use
+     */
+    labelComponent: React.PropTypes.element,
+    /**
```

Comment:
IMO it's a bad choice because then both `label` and `labelComponent` are potentially label instances – two props for the same thing, with unclear semantics if you supply both (or if you even can) – one prop just happens to get wrapped with `VictoryLabel` for you.

And yes, if we eliminated `label` then `labelComponent` would take an instance with the label text – I'm proposing that almost exactly, except we should ditch `labelComponent` in favor of `label`.